BuildAndTest.proj

```diff
@@ -9,12 +9,14 @@
     <SamplesSolution>$(MSBuildThisFileDirectory)src\Samples\Samples.sln</SamplesSolution>
     <TestVsi Condition="'$(testVsiNetCore)' == 'true'">true</TestVsi>
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <RunTestArgs>$(RunTestArgs) -timeout:50</RunTestArgs>
```

Comment:
This is short term only.  I have a different change where I put our mix of PS1, Batch and MSBuild into a single Powershell file.  Much easier to manage there.